### DISCUSSION TITLE
Command line installation of 9.19.0 does not work

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#16118] [Archive]
```
npm install directus@9.19.0 --save
npm ERR! code ETARGET
npm ERR! notarget No matching version found for directus@9.19.0.
```
and
`npm install directus --save
`
keeps on 9.18.1

**Comment by pupugit** [comment]
There was a problem with the build process. That is the reason why 9.19.0 is not available for npm.

**Comment by rijkvanzanten** [comment]
Yeah there have been some updates to GH actions recently that seemingly are causing some issues in the release flow now.. The NPM packages should be up to date as per usual, but we're still figuring out what to update to get Docker builds back online 👍🏻

**Comment by Pachat** [comment]
npm works with 9.19.2
